plugins/source/gandi/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/gandi/resources/plugin"
+	"github.com/cloudquery/plugin-sdk/serve"
+)
+
+// const sentryDSN = "https://6beffd303fbd4661a7b7c6c2d546b580@o1396617.ingest.sentry.io/6747634" TODO
```

Comment:
This requires updating once we create the new project.

Reply:
See this comment https://github.com/cloudquery/cloudquery/pull/4940#discussion_r1031644219